deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/codegen/ModelCodegenConfigParser.java

```diff
@@ -17,16 +18,19 @@
 public final class ModelCodegenConfigParser {
```

Comment:
Maybe we should reconsider renaming this class since now we are using it for the API generator as well. wdyt?

Reply:
i agree.  on it